ilastik/applets/dataSelection/dataSelectionApplet.py

```diff
@@ -124,209 +221,110 @@ def parse_known_cmdline_args(cls, cmdline_args, role_names):
-            num_missing = len(role_paths[0]) - len(role_paths[role_index])
-            role_paths[role_index] += [None] * num_missing
-        return role_paths
-
-    def configure_operator_with_parsed_args(self, parsed_args):
-        """
-        Helper function for headless workflows.
-        Configures this applet's top-level operator according to the settings provided in ``parsed_args``.
-
-        :param parsed_args: Must be an ``argparse.Namespace`` as returned by :py:meth:`parse_known_cmdline_args()`.
-        """
-        role_names = self.topLevelOperator.DatasetRoles.value
-        role_paths = self.role_paths_from_parsed_args(parsed_args)
-
-        for role_index, input_paths in list(role_paths.items()):
-            # If the user doesn't want image stacks to be copied into the project file,
-            #  we generate hdf5 volumes in a temporary directory and use those files instead.
-            if parsed_args.preconvert_stacks:
-                import tempfile
-
-                input_paths = self.convertStacksToH5(input_paths, tempfile.gettempdir())
-
-            input_infos = [FilesystemDatasetInfo(filepath=p) if p else None for p in input_paths]
-            if parsed_args.input_axes:
-                for info in [_f for _f in input_infos if _f]:
-                    info.axistags = vigra.defaultAxistags(parsed_args.input_axes)
-
-            opDataSelection = self.topLevelOperator
-            existing_lanes = len(opDataSelection.DatasetGroup)
-            opDataSelection.DatasetGroup.resize(max(len(input_infos), existing_lanes))
-            for lane_index, info in enumerate(input_infos):
-                if info:
-                    opDataSelection.DatasetGroup[lane_index][role_index].setValue(info)
-
-            need_warning = False
-            for lane_index in range(len(input_infos)):
-                output_slot = opDataSelection.ImageGroup[lane_index][role_index]
-                if output_slot.ready() and output_slot.meta.prefer_2d and "z" in output_slot.meta.axistags:
-                    need_warning = True
-                    break
-
-            if need_warning:
-                logger.warning(
-                    "*******************************************************************************************"
-                )
-                logger.warning(
-                    "Some of your input data is stored in a format that is not efficient for 3D access patterns."
-                )
-                logger.warning("Performance may suffer as a result.  For best performance, use a chunked HDF5 volume.")
-                logger.warning(
-                    "*******************************************************************************************"
-                )
+    def create_dataset_info(
+        self, url: Union[Path, str], axistags: Optional[vigra.AxisTags]=None, sequence_axis: str="z"
+    ) -> DatasetInfo:
+        url = str(url)
+        if isUrl(url):
```

Comment:
I agree. I think the appropriate thing to do would be to change `isUrl` to `isRemote`, since a URL can very well refer to a local path... In fact, the `UrlDatasetInfo` should probably be renamed to `RemoteDatasetInfo` too.

If you don't mind, I'd rather do this in another PR, though. It's an old function, which is used by a few other files in the project =/